### PR TITLE
Add mobile popup question to msal-browser FAQ

### LINKED ISSUE #5426: The msal-browser Popup API does not work on the mobile web browser.
### Core Library

MSAL.js v2 (@azure/msal-browser)

### Core Library Version

2.32.0

### Wrapper Library

Not Applicable

### Wrapper Library Version

None

### Public or Confidential Client?

Public

### Description

When using Vue sample app located [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-browser-samples/vue3-sample-app) to login via Popup API on mobile web browser it will not work as expected but when switching to redirect API it will work normally.

I've tested this using both Android and iPhone mobile browsers, and it's not working when using Popup API, but when using the Redirect API it will work as expected.

Is this a bug on the mobile device, or is this expected behavior?

### Error Message

_No response_

### Msal Logs

_No response_

### MSAL Configuration

```javascript
The same as in the Vue sample app.
(https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-browser-samples/vue3-sample-app).
```

### Relevant Code Snippets

```javascript
Running the Vue sample app as is
(https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-browser-samples/vue3-sample-app)
```

### Reproduction Steps

1. Cloned this [sample app](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-browser-samples/vue3-sample-app).
2. Modified msal config file to match my app registration.
3. Created `dist` folder by running `npm run build`
4. Deployed `dist` folder to azure static web and tested on iPhone web browser using [this link](https://redirecturiblank.z1.web.core.windows.net/).
7. Popup API not working for mobile web browser (Android and iPhone ).

### Expected Behavior

msal-browser Popup API to login and return token as Redirect API.

### Identity Provider

Azure AD / MSA

### Browsers Affected (Select all that apply)

Chrome, Safari, Firefox, Edge

### Regression

@azure/msal-browser 2.32.0

### Source

Internal (Microsoft)

### PR DESCRIPTION
**Pull request by hectormmg**
This PR adds a FAQ regarding popup API support on mobile browsers

### CODE REVIEW

File: lib/msal-browser/FAQ.md [hunk 1]
```diff
@@ -128,6 +129,10 @@ Yes, MSAL.js 2.x can be used as a custom authentication provider for the [Micros
 
 Yes, we recommend the new [Powershell Graph SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell) for doing so. For instance, the script below creates an Azure AD application with redirect URI of type **SPA** and **User.Read** permission for Microsoft Graph in a tenant specified by the user, and then provisions a service principal in the same tenant based on this application object:
 
+## Can I acquire tokens using popups on mobile browsers
+
+While MSAL's `loginPopup` and `acquireTokenPopup` APIs should work on mobile browsers that support popups, using popup APIs is discouraged on mobile scenarios, mainly because not all browsers support popups. We recommend using redirect APIs such as `loginRedirect` and `acquireTokenRedirect` when your application is running in a mobile browser.
```
Comment (tnorling) [inline lib/msal-browser/FAQ.md]:
```suggestion
While MSAL's `loginPopup` and `acquireTokenPopup` APIs may work on some mobile browsers, using popup APIs is discouraged on mobile devices as not all browsers will support this flow. We recommend using redirect APIs such as `loginRedirect` and `acquireTokenRedirect` when your application is running on a mobile device.
```

File: lib/msal-browser/FAQ.md [hunk 2]
```diff
@@ -128,6 +129,10 @@ Yes, MSAL.js 2.x can be used as a custom authentication provider for the [Micros
 
 Yes, we recommend the new [Powershell Graph SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell) for doing so. For instance, the script below creates an Azure AD application with redirect URI of type **SPA** and **User.Read** permission for Microsoft Graph in a tenant specified by the user, and then provisions a service principal in the same tenant based on this application object:
 
+## Can I acquire tokens using popups on mobile browsers
```
Comment (tnorling) [inline lib/msal-browser/FAQ.md]: Can we make this more general? i.e. "Does MSAL.js support mobile browsers?" and add a statement to the effect of "in general, yes ,however, given we do not test specifically test against mobile browsers support may be limited and some features and flows may not work. For problems specific to mobile browsers not listed here please open an issue with clear instructions to reproduce"

Comment (tnorling) [reply lib/msal-browser/FAQ.md]: And then move this to be right below the browser support question

Comment (derisen) [reply lib/msal-browser/FAQ.md]: +1

Comment (hectormmg) [reply lib/msal-browser/FAQ.md]: Updated, thanks for the feedback!

**Review by jo-arroyo — APPROVED** [review]

**Review by derisen — APPROVED** [review]

**Pull request merged**